db/migrate/20191120204541_add_government_id_to_edition.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddGovernmentIdToEdition < ActiveRecord::Migration[5.2]
+  def change
+    add_column :editions, :government_id, :integer
```

Comment:
You're right that this could fall out of sync and and we can't add any db constraints. I don't really think a validation is the best approach to deal with this though as this creates a situation where an external event can make other actions stop working, for example imagine editing the title blowing up because the government is wrong.

I don't think of this all as a huge concern as I see this value acting essentially as a cache of the last time this was synced which tells us what we think we told the Publishing API.